src/Capture/CaptureMode.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License (AFL 3.0)
+ */
+
+namespace PrestaShop\Module\PrestashopCheckout\Capture;
+
+/**
+ * Class CaptureMode used to get / update the capture mode from configuration
+ */
+class CaptureMode
+{
+    const PS_CHECKOUT_INTENT = 'PS_CHECKOUT_INTENT';
+
+    const CAPTURE = 'CAPTURE';
+
+    /**
+     * @param string $captureMode
+     */
+    public function update($captureMode)
+    {
+        \Configuration::updateValue(
```

Comment:
You can use new PrestaShopConfiguration service introduced in https://github.com/PrestaShopCorp/ps_checkout/pull/453

Reply:
done in next commit